doc/source/drivers/raster/vrt_processed_dataset.rst

```diff
@@ -12,6 +12,9 @@ to apply chained processing steps that may apply to several bands at the same ti
 The following built-in algorithms are introduced, and may typically be applied
 in the following order:
 
+- BandScaleOffset (since GDAL 3.11): apply per-band scale and offset values from
+  :cpp:func:`GDALRasterBand::GetScale` and :cpp:func:`GDALRasterBand::GetOffset`.
```

Comment:
an example would be welcome

Reply:
The example could describe how to set back the scale and offset values of the input data, which were "unscaled" earlier.
User may have data like the recommended DGIWG DEM variant that is scaled into Int16, and it may be surprise that data gets automatically unscaled, but it must be explicitly scaled back if Int16 with scale/offset is still desired.
I apologize if I have misunderstood something.

Reply:
> and it may be surprise that data gets automatically unscaled, but it must be explicitly scaled back if Int16 with scale/offset is still desired

I think a challenge with automatically re-scaling is knowing what scale factor to use. If we've applied an arbitrary processing algorithm to the inputs, I don't think we can assume the input scale/offset are still appropriate.